tools/database/src/memtrie.rs

```diff
@@ -269,6 +272,82 @@ impl<'a, 'b> MemtrieSizeCalculator<'a, 'b> {
             }
         }
 
-        Ok(total_size)
+        Ok(ByteSize::b(total_size))
+    }
+}
+
+#[derive(clap::Parser)]
+pub struct FindBoundaryAccountCommand {
+    #[clap(long, help = "ID of the shard to split")]
+    shard_uid: ShardUId,
+}
+
+impl FindBoundaryAccountCommand {
+    pub fn run(
+        &self,
+        home: &Path,
+        genesis_validation: GenesisValidationMode,
+    ) -> anyhow::Result<()> {
+        let env_filter = EnvFilterBuilder::from_env().verbose(Some("memtrie")).finish()?;
```

Comment:
Is it necessary to force verbosity for a specific module? Can't users `env RUST_LOG=memtrie` as needed themselves?